src/style.css

```
@@ -1 +1,81 @@
 /* Styles go here */
+html {
+  background-color: hsl(0, 0%, 90%);
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  max-width: 1200px;
+  min-width: 1024px;
+  background-color: white;
+  margin: auto;
+  padding: 0 50px;
+}
+
+.nav,
+.header__logo {
+  display: flex;
+  align-items: center;
+}
```

```suggestion
  padding: 0 50px;
  align-items: center;
}
```
looks like redundant styles for nav and logo.
instead set align-items: center for .header

---

ok. but my .header__logo must  be display:flex